HopsanGenerator/include/GeneratorTypes.h

```diff
@@ -25,34 +25,73 @@
 #ifndef GENERATORTYPES_H
 #define GENERATORTYPES_H
 
+#include "hopsangenerator_win32dll.h"
+
 #include <QString>
 #include <QStringList>
 #include <QList>
-//#include <QVector>
 
-#include "GeneratorUtilities.h"
+namespace hopsan {
 
+namespace os_strings {
+    constexpr auto win = "win";
+    constexpr auto win32 = "win32";
+    constexpr auto win64 = "win64";
+    constexpr auto apple = "apple";
+    constexpr auto Linux = "linux";
```

Comment:
Why capital letter on Linux but not on the others?

Reply:
I will fix this later, the code needs some reworking anyway.